static/bidder-info/driftpixel.yaml

```diff
@@ -13,7 +13,6 @@ capabilities:
       - video
       - native
 userSync:
-  # DriftPixel supports user syncing, but requires configuration by the host. contact this
-  # bidder directly at the email address in this file to ask about enabling user sync.
-  supports:
-    - redirect
+  redirect:
+    url: "https://sync.driftpixel.live/psync?t=s&e=0&gdpr={{.GDPR}}&consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&cb={{.RedirectURL}}"
```

Comment:
it works 
![image](https://github.com/user-attachments/assets/b30f291b-dfdc-47e6-b702-3eb3f928435d)

Reply:
![image](https://github.com/user-attachments/assets/1f3ac9a1-f6be-4f4f-aba5-4c3df7d95b28)